### PR TITLE
Feeder

### PR DESCRIPTION
**Pull request by vmad18**

### CODE REVIEW

File: src/main/kotlin/com/team4099/robot2021/config/ControlBoard.kt [hunk 1]
```diff
@@ -2,60 +2,57 @@ package com.team4099.robot2021.config
...
+    get() = operator.dPadLeft || operator.dPadRight
 
-    val runFeederIn: Boolean
-        get() = operator.dPadDown
+  val slowMode: Boolean
+    get() = driver.dPadDown
 
-    val runFeederOut: Boolean
-        get() = operator.dPadUp
+  val runFeederIn = Trigger{ operator.dPadDown};
```
Comment (rbhog) [inline src/main/kotlin/com/team4099/robot2021/config/ControlBoard.kt:52]: To confirm, you guys were only concerned with changing the feeder controls to triggers right?

**Review by rbhog — COMMENTED** [review]
Everything looks fine, just wanted to make sure that you were only concerned with making the triggers for the feeder. Everything else was the same as last year in ControlBoard

**Review by rbhog — APPROVED** [review]
Nice job!

**Review by rbhog — CHANGES_REQUESTED** [review]
We still need logging

File: src/main/kotlin/com/team4099/robot2021/commands/feeder/FeederCommand.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.team4099.robot2021.commands.feeder
+import com.team4099.robot2021.config.subsystems.Feeder
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class FeederCommand(var dir: Feeder.FeederState): CommandBase(){
+  init{
+    addRequirements(Feeder)
+  }
```
Comment (eckshn) [inline src/main/kotlin/com/team4099/robot2021/commands/feeder/FeederCommand.kt]: Add events

**Review by rbhog — APPROVED** [review]
Nice Job!

**Pull request merged**